demo/marked-highlight.html

```diff
@@ -0,0 +1,33 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../marked-import.html">
+
+<link rel="import" href="./md-styles.html">
+
+<dom-module id="marked-highlight">
+
+  <!-- load assets for highlight-js -->
+  <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+
+  <template>
+    <style include="md-styles"></style>
+    <marked-element id="syntaxHighlight" on-syntax-highlight="_highlightSyntax">
+      <div slot="markdown-html"></div>
+      <script type="text/markdown" src="./example.md">
+        Loading markdown...
+      </script>
+    </marked-element>
+  </template>
+
+  <script>
+    class MarkedHighlight extends Polymer.Element {
```

Comment:
Unfortunately this repo is still a hybrid element. Can you please write this in hybrid syntax? i.e. `Polymer({...})`